Makefile

```diff
@@ -41,22 +41,33 @@ sync-googleapis:
 	rm -rf $(DEST)
 
 clean::  ## Remove generated files
+	#TODO this breaks things, undo when settled on dir structure
 	rm -rf testdata/conformance/*
 
 .PHONY: sync sync-googleapis
 
 # --- Protos -----------------------------------------------------------
 COMPILER_PROTO_FILES = $(wildcard compiler/testdata/*.proto)
 COMPILER_PB_FILES = $(patsubst compiler/testdata/%.proto,compiler/testdata/pb/%.pb,$(COMPILER_PROTO_FILES))
+CONFORMANCE_PROTO_FILES = $(wildcard testdata/conformance/*.proto)
+CONFORMANCE_PB_FILES = $(patsubst testdata/conformance/%.proto,testdata/conformance/pb/%.pb,$(CONFORMANCE_PROTO_FILES))
 
-pb: $(COMPILER_PB_FILES)  ## Generate binary FileDescriptorSet as pb files from compiler/testdata/*.proto
+pb:  | testdata/conformance/pb $(COMPILER_PB_FILES) $(CONFORMANCE_PB_FILES)
+## Generate binary FileDescriptorSet as pb files from compiler/testdata/*.proto
 
 compiler/testdata/pb/%_no_include.pb: compiler/testdata/%_no_include.proto
 	protoc -I compiler/testdata  -o $@ $<
 
 compiler/testdata/pb/%.pb: compiler/testdata/%.proto
 	protoc --include_imports -I compiler/testdata -o $@ $<
 
+testdata/conformance/pb:
+	mkdir -p testdata/conformance/pb
+
+# TODO: also do unit tests from google/protobuf
+testdata/conformance/pb/%.pb: testdata/conformance/%.proto
```

Comment:
Sorry about my last comment. You should move the order-only prerequisite to this rule